content/pt-br/docs/concepts/security/windows-security.md

```diff
@@ -0,0 +1,56 @@
+---
+reviewers:
+  -
+title: Segurança para Nós Windows
+content_type: concept
+weight: 40
+---
+
+<!-- overview -->
+
+Esta página descreve considerações de segurança e boas práticas específicas para o sistema operacional Windows.
+
+<!-- body -->
+
+## Proteção para dados Secret nos Nós
+
+No Windows, os dados do Secret são escritos em texto claro no Nó local do
+armazenamento (em comparação ao uso de tmpfs / in-memory filesystems no Linux). Como um cluster
```

Comment:
Suggestion: "sistemas de arquivo em memória" instead of "in-memory filesystems".